Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -256,41 +283,30 @@ nrow(min_psm_count_input_rbp)
 setdiff(rownames(data_to_model),
         min_psm_count_input_rbp$Master.Protein.Accessions)
 
+psm_counts <-input_min_psm_count %>% tibble::remove_rownames() %>%
+  tibble::column_to_rownames('Master.Protein.Accessions') 
 
+limma_fit$count = psm_counts[common_proteins,'min_psm_count']
 ```
 
 
 
 ```{r}
-#take common proteins between input and OOPS protein Rds and input min psm count, use this object for deqms
```

Comment:
Now we have access to the `limma_fit` object from above, we can use that without re-running the limma steps